src/components/challengeCard.tsx

```diff
@@ -0,0 +1,52 @@
+"use client";
+import Text from "~/components/text";
+import { BsNut } from "react-icons/bs";
+
+import React from "react";
+import { useState } from "react";
+
+
+
+// Main Challenge Card Component
+const ChallengeCard: React.FC<{
+  title: string; type: string;
+  description: string;
+  points: number;
+  difficulty: string;
+  solves: number;
+}> = ({ title, type, description, points, difficulty, solves = 0 }) => {
+  const [isHovered, setIsHovered] = useState(false);
+  {
+    return (
+      <div
+        className={`p-6 rounded-lg shadow-lg transition-opacity duration-300 cursor-pointer mb-5 ${isHovered ? "opacity-80" : "opacity-100"
```

Comment:
remove `transition-opacity`
add `transition-bg duration-200 ease-in-out`

Reply:
done